sdk/python/symbolchain/facade/SymbolFacade.py

```diff
@@ -32,6 +33,7 @@ class SymbolFacade:
 	Address = Address  # pylint: disable=duplicate-code
 	KeyPair = KeyPair  # pylint: disable=duplicate-code
 	Verifier = Verifier  # pylint: disable=duplicate-code
+	SharedKey = SharedKey
```

Comment:
hmm, so seems we actually only need (X-5), where X is number of common lines...
so in case above we actually only need on Address and a KeyPair, so not sure if it's better to delete the one on Verifier or add additional one on SharedKey...

Reply:
can delete if that works 👍